src/reducers/profile-view.js

```diff
@@ -867,3 +868,97 @@ export const selectedThreadSelectors: SelectorsForThread = (() => {
   const result2: SelectorsForThread = result;
   return result2;
 })();
+
+export type SelectorsForNode = {
+  getName: State => string,
+  getIsJS: State => boolean,
+  getLib: State => string,
+  getTimingsForSidebar: State => TimingsForPath,
+};
+
+export const selectedNodeSelectors: SelectorsForNode = (() => {
+  const getName = createSelector(
+    selectedThreadSelectors.getSelectedCallNodePath,
+    selectedThreadSelectors.getFilteredThread,
+    (selectedPath, { stringTable, funcTable }) => {
+      if (!selectedPath.length) {
+        return '';
+      }
+
+      const funcIndex = ProfileData.getFuncIndex(selectedPath);
+      return stringTable.getString(funcTable.name[funcIndex]);
+    }
+  );
+
+  const getIsJS = createSelector(
+    selectedThreadSelectors.getSelectedCallNodePath,
+    selectedThreadSelectors.getFilteredThread,
+    (selectedPath, { funcTable }) => {
+      if (!selectedPath.length) {
+        return false;
+      }
+
+      const funcIndex = ProfileData.getFuncIndex(selectedPath);
+      return funcTable.isJS[funcIndex];
+    }
+  );
```

Comment:
I implemented this function for an earlier version of my patch but don't use it now. I thought it could be eventually useful so I left it here, but please tell me if you prefer that I remove it.

Reply:
Sure, let's leave it there.